.gitignore

```diff
@@ -1,3 +1,5 @@
-.idea/*
+.idea
 *.iml
-target/*
+target
+src/main/resources
```

Comment:
let's remove this line
we should not exclude `src/main/resources` from being pushed to github